Aboutme.md

```diff
@@ -0,0 +1,19 @@
+### :open_file_folder: What have I learnt?
+* The different principles of Software Development.
+* How to compile a Java programme.
```

Comment:
Tends to be written using the Americanised spelling, e.g. 'Java program', 'computer program', etc.